pkg/machine/e2e/list_test.go

```diff
@@ -135,6 +135,13 @@ var _ = Describe("podman machine list", func() {
 		var listResponse []*machine.ListReporter
 		err = jsoniter.Unmarshal(listSession.Bytes(), &listResponse)
 		Expect(err).To(BeNil())
+
+		// table format includes the header
+		list := new(listMachine)
+		listSession, err := mb.setCmd(list.withFormat("table {{.Name}}")).run()
+		Expect(err).NotTo(HaveOccurred())
+		Expect(listSession).To(Exit(0))
+		Expect(len(listSession.outputToStringSlice())).To(Equal(2))
```

Comment:
Using `len()` is deprecated: it leads to worthless error messages like "expected 1 to equal 2". Could I ask you to change this (and like 122) to:
```go
    Expect(listSession.outputToStringSlice()).To(HaveLen(2))
```
This will show an actually useful error like 'expected "this is the string" to have length 2'. And could you add a test for `To(ContainSubstring("NAME"))` here also?

This is in no way your fault - you just copied someone else's broken code. I'm commenting because I hope you'll take the opportunity (and, sigh, the CI cycles) to tidy things up.